help2man/help2man.inc

```diff
@@ -0,0 +1,9 @@
+# -*- mode:python; -*-
+RECIPE_TYPES = "machine native"
+
+EXTRA_AUTORECONF="-Iaclocal.m4"
+inherit autotools-autoreconf
```

Comment:
Did you find time to test if we could stay with just autotools instead of autotools-autoreconf?